.jshintignore

```diff
@@ -0,0 +1,2 @@
+js/skip-link-focus-fix.js
+**.min.js
```

Comment:
Why are you excluding `skip-link-focus-fix.js`? I do not think that `**.min.js` is necessary here.